hare/hare.go

```diff
@@ -213,19 +225,28 @@ var (
 )
 
 // GetResults returns the hare output for a given LayerID. returns error if we don't have results yet.
-func (h *Hare) GetResult(id types.LayerID) ([]types.BlockID, error) {
-	if h.isTooLate(InstanceId(id)) {
+func (h *Hare) GetResult(lower types.LayerID, upper types.LayerID) ([]types.BlockID, error) {
+
+	if h.isTooLate(InstanceId(lower)) {
 		return nil, ErrTooOld
 	}
 
+	var results []types.BlockID
 	h.mu.RLock()
-	blks, ok := h.outputs[id]
-	if !ok {
-		h.mu.RUnlock()
-		return nil, ErrTooEarly
+	for id := lower; id <= upper; id++ {
+		blks, ok := h.outputs[id]
+		if !ok {
+			continue
```

Comment:
is this expected? log it?

Reply:
i think its redundant we have logs on stuck hare's and we have the results in the blocks
if we think theres an issue with the maAtuf and the buffer then maybe....